### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/) for ChainSafe Files/Chainsafe Files user interface.

Current translation status:

Weblate translation status

### CODE REVIEW

File: packages/files-ui/src/locales/fr/messages.po [hunk 1]
```diff
@@ -709,6 +710,8 @@ msgstr "Sauvegardé par le mineur"
 
 msgid "System maintenance is scheduled to start at {0}. The system will be unavailable."
 msgstr ""
+"La maintenance du système est prévue pour démarrer à {0}. Le système sera "
```
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
"Une maintenance du système est prévue pour démarrer à {0}. Le système sera "
```

File: packages/files-ui/src/locales/fr/messages.po [hunk 2]
```diff
@@ -832,6 +835,9 @@ msgstr "Vous voulez participer à l'élaboration de ce produit ?"
 
 msgid "We are performing routine maintenance of the system. Service status updates will be posted on the <0>Files Support Channel</0> on Discord"
 msgstr ""
+"Nous effectuons une maintenance de routine du système. Les mises à jour de "
+"l'état du service seront postées sur le <0>Canal de prise en charge des "
+"fichiers</0> sur Discord"
```
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
"l'état du service seront postées sur le canal <0>Files Support"
"fichiers</0> sur Discord"
```

File: packages/files-ui/src/locales/fr/messages.po [hunk 3]
```diff
@@ -832,6 +835,9 @@ msgstr "Vous voulez participer à l'élaboration de ce produit ?"
 
 msgid "We are performing routine maintenance of the system. Service status updates will be posted on the <0>Files Support Channel</0> on Discord"
 msgstr ""
+"Nous effectuons une maintenance de routine du système. Les mises à jour de "
+"l'état du service seront postées sur le canal <0>Files Support"
+"fichiers</0> sur Discord"
```
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
"</0> sur Discord"
```

**Review by Tbaut — APPROVED** [review]

**Comment by asnaith** [comment]
I'm making the assumption @Tbaut knows French very well, if he's good with this then so am I 😄

**Review by asnaith — APPROVED** [review]

File: packages/files-ui/src/locales/fr/messages.po (hunk 1 above)
Comment (Tbaut) [reply packages/files-ui/src/locales/fr/messages.po]:
```suggestion
"Une maintenance du système est prévue pour démarrer à {0}. Le système sera "
```

Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
"Une maintenance du système est prévue pour démarrer à {0}. Le système sera "
```

File: packages/files-ui/src/locales/fr/messages.po (hunk 2 above)
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
"l'état du service seront postées sur le canal <0>Files Support<0> sur Discord"
```

**Comment by Tbaut** [comment]
For once, the translation was pretty bad :)

**Pull request merged**